ci/benchmark_parser.py

```diff
@@ -104,19 +161,20 @@ def recursive_parse(directory, walk_subdirs=False, name_suffix="", compute_throu
                         "latency",
                         operator,
                         params,
-                        display_name=display_name
+                        display_name=display_name,
                     )
                 )
 
                 # This is a special case where PBS are blasted as vector LWE ciphertext with
                 # variable length to saturate the machine. To get the actual throughput we need to
                 # multiply by the length of the vector.
-                if "PBS_throughput" in test_name and "chunk" in test_name:
+                if "pbs_throughput" in test_name.lower() and test_name.lower().endswith("chunk"):
                     try:
-                        multiplier = int(test_name.split("chunk")[0].split("_")[-1])
+                        multiplier = int(test_name.strip("chunk").split("::")[-1])
                     except ValueError:
```

Comment:
I'm going to be annoyin line 173 does not have the lower

so let's do

test_name = test_name.lower() and keep the rest as before ?